web/static/js/display.js

```diff
@@ -0,0 +1,89 @@
+const div = document.querySelector('#printDiv');
+var lastRound = "0";
+
+function testFetchKeyAndPublic() {
+  move();
+  var date = new Date();
+  var timestamp = date.toISOString();
+
+  identity = {Address: "127.0.0.1:8000", TLS: false}
```

Comment:
This feels like it should be an argument. Would it be possible to move it to the HTML file ?
As well, instead of having two different calls, you could have only one in the HTML file, stg like:
```
beaconRound(identity, key)
```
If `key == ""` then you have to fetch the key. That would allow a user to only change the address (and potentially the key) inside the `index.html` which is the central where we want to look for. As well, for the future, it will enable you to easily shift the server to contact when the user clicks on a server from the list.

Reply:
I can move it yes. `display.js` is not part of DRandJS so the whole code could be moved to the html actually.  
And good idea for merging the two functions, I'll do that

Reply:
No need to move the whole, separation of concern is good ;) Just the address / key args.